### ISSUE TITLE
Marker Normalization

### ISSUE THREAD

**Issue by akurmustafa** [mahmoodlab/KRONOS#3]
Hi thanks for this awesome work. I have couple of questions regarding the usage of the model:
- You provided mean and std values for the 175 markers used in the training in the [metadata](https://huggingface.co/MahmoodLab/KRONOS/blob/main/marker_metadata.csv) file. How should I normalize the data if the marker is not one of the 175 pre-trained markers?
- In the paper, for prostate cancer evaluation you have used `Androgen Receptor`. However, I cannot see the `Androgen Receptor` among the listed 175 markers. Which marker corresponds to `Androgen Receptor` among the listed 175 markers?
- Should we just do the `(img-mean)/std` in the spatial proteomics data without any other normalization? I have seen some people do other normalizations (such as clipping pixel intensity values to lower_percentile=1 and upper_percentile=99) to the spatial proteomics data. What is the recommended approach for preprocessing marker data?

Best

**Comment by m-shaban** [comment]
Thank you for your interest in our work!

1. **Normalization for markers not among the 175 pre-trained markers**:
   If your marker is not among the 175 used in training, you can calculate the mean and standard deviation from your own dataset. In our experience, using dataset-specific statistics did not lead to significant changes in model performance.

2. **Use of Androgen Receptor in the paper**:
   The model is **marker-agnostic**; therefore, you can assign any unused marker ID for markers not included in the original 175, including Androgen Receptor. We followed the same approach. You can refer to this guide on [how marker IDs are assigned](https://github.com/mahmoodlab/KRONOS/blob/main/tutorials/1%20-%20Data-Download-And-Preprocessing.ipynb).

3. **Normalization procedure for spatial proteomics data**:
   Before applying `(img - mean) / std` normalization, you need to first scale the image values to a 0–1 range. This is done by dividing the image by the maximum possible value based on its data type — for example:

   * Use `255.0` for `uint8` images
   * Use `65535.0` for `uint16` images

   This max value should be specified in the config using the `"marker_max_values"` parameter. For example:

   ```json
   "marker_max_values": 65535.0 
   ```
